WPMedia/fusion-news-theme-blocks
# THEMES-1219: Logical Properties | Lead art and Video player blocks

## Description

This PR updates the logical properties on the masthead and footer blocks. The links bar block does not need any changes.

## Jira Ticket

- [THEMES-1219](https://arcpublishing.atlassian.net/browse/THEMES-1219)

## Acceptance Criteria

1. The above logical properties are applied and tested on the lead art and video player blocks.
2. Storybook is updated with RTL. 

## Test Steps

1. Checkout this branch `git checkout THEMES-1219`
3. Run fusion repo with linked blocks `npx fusion sart -f -l @wpmedia/lead-art-block,@wpmedia/video-player-block`
4. Inspect the lead art and video player blocks and see that they are using logical properties in the correct places.
5. It should match the designs in Figma.
6. Set `"textDirection": "rtl"` in the `"siteProperties"` object of the `blocks.json`.
7. Restart PageBuilder and verify that the styling adapts to the new text direction.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1219]: https://arcpublishing.atlassian.net/browse/THEMES-1219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
